data-engineering/How-to-use-conception-tickets.s.md

```diff
@@ -0,0 +1,26 @@
+# [Standard] How to tackle tickets with conception phase (3 min)
+
+### Owner: [Quentin Febvre](https://github.com/quentinf00)
+
+
+## Why:
+
+ - To detect problem earlier during the implementation of a ticket
+ - To ensure vsibility on what is done during the implementation of a ticket
+
+This can be a tool to reduce the lead time on features with uncertainties, the conception phase can be used to prioritize kanban ticket into you sprint
+
+
+## Checks
+
+ - I have detailed the ticket in tasks smaller than 15min
+ - The tasks have been challenged by another dev of the team
+ - The dev tracks the time spent on each tasks using this tool: https://theodo.github.io/workflow-monitor
```

Comment:
Et si t'as pas Trello comme sur RSolve ?

Reply:
Good point, you can do your conception phase on your jira card, but you need to be extra rigorous about time tracking on each task. You can ask some co dev to help you track your time